src/main/java/com/auth0/client/mgmt/filter/PageFilter.java

```diff
@@ -32,4 +32,37 @@ public PageFilter withTotals(boolean includeTotals) {
         return this;
     }
 
+    /**
+     * Include the {@code from} parameter to specify where to start the page selection. Only applicable for endpoints that
+     * support checkpoint pagination.
+     * <p>
+     * <strong>Note: If this or the {@linkplain PageFilter#withTake(int)} is specified, any offset paging parameters set
+     * via {@linkplain PageFilter#withPage(int, int)} or {@linkplain PageFilter#withTotals(boolean)} will be disregarded
+     * by the API.</strong>
+     * </p>
+     * @param from the ID from which to start selection. This can be obtained from the {@code next} field returned from
+     *             a checkpoint-paginated result.
+     * @return this filter instance.
+     */
+    public PageFilter withFrom(String from) {
```

Comment:
Would it make sense to merge this with the "take" param, given that both are used for "checkpoint pagination"?

i.e. see above the `withPage(int pageNumber, int amountPerPage)`.

Reply:
The "from" param is not required, as the first call wouldn't include it, but subsequent calls would set this param using the `next` value from the paged response. So if they are combined, developers would need to pass a `null` value for the "from" param, which seems more awkward than specifying them separately IMO